src/main/webapp/scripts/user-script.js

```diff
@@ -433,8 +497,8 @@ function visualizeUser(userData, htmlid){
     // append the svg object to the body of the page
     // appends a 'group' element to 'svg'
     // moves the 'group' element to the top left margin
-    var str = "#" + htmlid;
-    var svg = d3.select(str).append("svg")
+    var strr = "#" + htmlid;
```

Comment:
if this is not used anywhere else except directly below the line below can be rewritten as 
`d3.select( "#" + htmlid).append("svg")`
or rename the variable to something more descriptive